src/features/usage-consolidation-accordion/summary-table/summary-table.js

```diff
@@ -62,85 +49,9 @@ const SummaryTable = ({
     usage,
   } = data?.attributes?.analysis;
 
-  const handleViewTitles = (onToggle) => () => {
-    onViewTitles();
-    onToggle();
-  };
-
-  const formatter = {
-    ucActions: () => (
-      <Dropdown
-        id="summary-table-actions-dropdown"
-        renderTrigger={({ onToggle, triggerRef, ariaProps, keyHandler, getTriggerProps }) => (
-          <DropdownButton
-            id="usage-consolidation-actions-dropdown-button"
-            ref={triggerRef}
-            onKeyDown={keyHandler}
-            marginBottom0
-            onClick={onToggle}
-            {...ariaProps}
-            {...getTriggerProps()}
-          >
-            <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions" />
-          </DropdownButton>
-        )}
-        renderMenu={({ onToggle }) => (
-          <DropdownMenu
-            role="menu"
-          >
-            {entityType === entityTypes.PACKAGE ? (
-              <Button
-                id="summary-table-actions-view-titles"
-                buttonStyle="dropdownItem fullWidth"
-                role="menuitem"
-                onClick={handleViewTitles(onToggle)}
-                marginBottom0
-              >
-                <Icon
-                  icon="eye-open"
-                  size="small"
-                >
-                  <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions.view" />
-                </Icon>
-              </Button>
-            ) : null
-            }
-            <div>
-              <Button
-                buttonStyle="dropdownItem fullWidth"
-                role="menuitem"
-                onClick={() => {
-                  onExportTitles(true);
-                  onToggle();
-                }}
-                disabled={isExportDisabled}
-                marginBottom0
-              >
-                <Icon
-                  icon="download"
-                  size="small"
-                >
-                  <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions.export" />
-                </Icon>
-              </Button>
-              {isExportDisabled && (
-                <span className={style['limit-error']}>
-                  <FormattedMessage id="ui-eholdings.usageConsolidation.summary.exportTitles.limit" />
-                </span>
-              )}
-            </div>
-          </DropdownMenu>
-        )}
-      />
-    )
-  };
-
   const customPropertiesWithFormatter = {
     ...customProperties,
-    formatter: {
-      ...(customProperties.formatter || {}),
-      ...formatter,
-    },
+    formatter: { ...(customProperties.formatter || {}) },
```

Comment:
We don't need this `customPropertiesWithFormatter` object anymore if we're not changing `formatter` property here

Reply:
That's make sense, fixed!